CELLULAR.mkd

```diff
@@ -0,0 +1,89 @@
+## CONFIGURATION OPTIONS SPECIFIC TO THE CELLULAR C5
```

Comment:
These docs should go in the `docs` directory and be added to one of the indexes, maybe from the advanced configuration options page - that way it will be available at vi-firmware.openxcplatform.com